forc/GFS/README.md

```diff
@@ -0,0 +1,94 @@
+## Description
+
+* This package generates DATM input data for the forcing fluxes at the sea surface from the NCEP operational GFSv16 outputs.
+
+## Inputs of conversion in conv_gfs2datm_long_beta4.f
+
+* INPUT Sources : from the output of GFSv16,
+
+I. ./gdas.${dat}/${hh}/atmos/gdas.t${hh}z.sfluxgrbf000.grib2 :: Input data are from the GFSv16 outs of initial analysis fields, 
```

Comment:
The data is from GFSv16, its said on line 7.  Is there a need to repeat it here.

Reply:
Cleaned it.